### ISSUE TITLE
1.12.2 Chisle Blocks Invisible

### ISSUE THREAD

**Issue by Trinii** [Soartex/Issues#21]
https://www.curseforge.com/minecraft/texture-packs/soartex-fanver-modded/files

I downloaded the Version from Dec 1, 2019 and there are a few Invisible Blocks. 
![2020-05-15_21 04 19](https://user-images.githubusercontent.com/65419600/82087624-c3d42880-96f0-11ea-90aa-06c63bbcd5fb.png)
![2020-05-15_21 00 29](https://user-images.githubusercontent.com/65419600/82087625-c59dec00-96f0-11ea-8107-cf8d2d98d473.png)

I'll show you 2 from them But there are much more like you can see on the Screenshot. 
Also some chisled Glass like (shale) #6037/8
Is there any way to fix them?

**Comment by louis71barker** [comment]
Im having this exact same issue
Was there a fix for this?

**Comment by Shoeboxam** [comment]
I was able to fix this by updating the ctm mod to the latest version. What is your CTM mod version?

**Comment by tartletboy** [comment]
Yeah I was able to reproduce this issue. Anything I can do to help diagnose the issue?

**Comment by Shoeboxam** [comment]
> Yeah I was able to reproduce this issue. Anything I can do to help diagnose the issue?

Could you provide your CTM mod version, and then attempt to upgrade the CTM mod and check if the issue has been resolved?
